### PR TITLE
Sync with usrse on May 16th 2021

### PR DESCRIPTION
**Pull request by vsoch**
This will update the content here to match usrse, something I can do but will get increasingly harder as time goes on!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,13 +328,6 @@ There are several reasons to do this:
  2. we can scope a particular kind of test to a service. For example, one service might just be to test the core software, another might be to build and deploy containers, and a third might be to preview a site.
  3. each CI service offers unique features. For example, GitHub has the closest integration with the repository here, and CircleCI allows us to preview artifacts.
 
-## Thanks
```
Comment (cosden) [inline README.md:331]: I think we want to keep this 'Thanks' section, right? It's part of the new design.

**Review by cosden — COMMENTED** [review]
Thanks for doing this @vsoch! I know it's going to get hard so doing it every couple weeks makes a lot of sense. 

Let's do what we can to only add jobs, events, and posts to the old site. That will minimize the changes and keep things fairly well contained.

**Comment by cosden** [comment]
Oops saw you just merged as I was reviewing. I think it's good, just that one Thanks section, which could always be added at the end.

File: README.md (hunk 1 above)
Comment (vsoch) [reply README.md:331]: haha oops, totally forgot about that. And I actually did check! I'll add it back now.

**Pull request merged**